packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings/pipeline-name.tsx

```diff
@@ -45,7 +45,7 @@ export const PipelineName: React.FunctionComponent<PipelineNameProps> = ({
           <Body>{name}</Body>
         </Tooltip>
       )}
-      {isModified && <span className={modifiedStyles}> - modified</span>}
+      {isModified && <span className={modifiedStyles}>&nbsp;– modified</span>}
```

Comment:
Is it intentional to add &nbsp; only on the left side?

Reply:
Yeah! need an empty space between Title and modified flag (`pipeline - modified`). And while rendering, the white-space is not shown.